### ISSUE TITLE
og:image:alt and twitter:image:alt are missing

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#3563]
**Describe the defect**
Open Graph and Twitter meta data for images should have alt text. 

**To Reproduce**
1. Go to [an event node](https://www.va.gov/outreach-and-events/events/virtual-coffee-house-veteran-business-chat) or a location node such as [University Drive](https://www.va.gov/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/), which have images. 
2. Inspect the meta
3. Note that twitter:image and og:image are specified, but not their respective alt text

**Expected behavior**
1. Alt text should be included

**Screenshots**
You can also use an og or twitter meta debugger. Here's[ one from FB](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fwww.va.gov%2Fpittsburgh-health-care%2Flocations%2Fpittsburgh-va-medical-center-university-drive%2F) (requires login). 
![Sharing_Debugger_-_Facebook_for_Developers](https://user-images.githubusercontent.com/643678/100010175-dd3fe680-2d9d-11eb-8cc6-065ff7f586aa.jpg)
